sim/SIM-PROTOCOL.md

```diff
@@ -670,10 +672,11 @@ To be exact, `CHOICE` is one of:
 Once a choice has been set for all players who need to make a choice, the
 battle will continue.
 
-All decisions except `/undo` can be sent with `|RQID` at the end. `RQID` is
-`REQUEST.rqid` from `|request|`. Each `RQID` is a unique number used to
-identify which action the request was intended for and is used to protect
-against race conditions involving `/undo` (the cancel button).
+All decisions except `/undo` can be sent with `|RQID` at the end, where `RQID`
+can be computed from the number of `|request|`'s received (eg. the fifth
+`|request|` is `RQID` 5). The `RQID` is used to identify which action the
```

Comment:
This part still mentions RQID.

I think it might make sense to document RQID in SIM-PROTOCOL, and just note that it doesn't appear if you're using the sim directly rather than connecting through the server.

Reply:
It deliberately mentions rqid, but it explains that its computable not that its part of request.

TBH though, im beyond caring about this, I was just trying to do some due diligence followup. If you or anyone else wants to make the documentation more clear I'd be happy, but this is not worth it to me at this point.

Reply:
Sure.